.github/workflows/buildtest.yml

```diff
@@ -16,17 +16,17 @@ jobs:
   gcc13_assertions_build:
     runs-on: ubuntu-latest
     container:
-      image: ghcr.io/4c-multiphysics/4c-dependencies-ubuntu24.04:da17023e
+      image: ghcr.io/4c-multiphysics/4c-dependencies-ubuntu24.04:bbac1cd9
       options: --user root --env OMPI_ALLOW_RUN_AS_ROOT=1 --env OMPI_ALLOW_RUN_AS_ROOT_CONFIRM=1
     defaults:
       run:
         shell: bash
     steps:
       - uses: actions/checkout@v4
       - name: Check docker hash
-        uses: ./.github/actions/compute-and-check-dependencies-hash
+        uses: ./.github/actions/compute-dependencies-hash
         with:
-          docker_image_hash: $DEPENDENCIES_HASH
+          assert_running_in_correct_image: 'true'
```

Comment:
Does it make sense to flip the default to true? The input line can easily be forgotten which would mean that we do not check the hash. If the default is true, forgetting the line is just annoying but causes no harm. WDYT?

Reply:
I just flipped the defaults because of the name. With the name `compute-dependencies-hash` I would not expect to check the hash. But I agree, checking by default is more safe. I will change the default (and maybe the name)